scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1760,10 +1762,21 @@ class Router(formatOps: FormatOps) {
       case FormatToken(c: T.Comment, _, _) =>
         Seq(Split(getMod(formatToken), 0))
 
-      case FormatToken(_: T.KwImplicit, _, _)
+      case FormatToken(_: T.KwImplicit | T.Ident("using"), _, _)
           if !style.binPack.unsafeDefnSite &&
-            !style.verticalMultiline.atDefnSite =>
-        opensImplicitParamList(prevNonComment(prev(formatToken))).fold {
+            !style.verticalMultiline.atDefnSite && (prev(formatToken) match {
+              // check if using is the soft kw
+              case FormatToken(_: T.LeftParen, T.Ident("using"), _) => true
+              case FormatToken(_, _: T.KwImplicit, _) => true
+              case _ => false
+            }) =>
+        val argsOrParamsOpt = formatToken.meta.leftOwner match {
+          // for the using argument list
+          case _: ApplyUsing => Some(getApplyArgs(formatToken, false)._2)
```

Comment:
```suggestion
          case _: ApplyUsing => Some{
            val (_, args) = getApplyArgs(formatToken, false)
            args
          }
```
maybe? But this is just a nitpick for sure,